nemo/collections/multimodal/modules/stable_diffusion/diffusionmodules/openaimodel.py

```diff
@@ -1242,13 +1245,10 @@ def _load_pretrained_model(self, state_dict, ignore_mismatched_sizes=False, from
         ):
             # GroupNormOpt fuses activation function to one layer, thus the indexing of weights are shifted for following
             for key_ in missing_keys:
-                try:
-                    s = key_.split('.')
-                    idx = int(s[-2])
-                    new_key_ = ".".join(s[:-2] + [str(int(idx + 1))] + [s[-1]])
-                    state_dict[key_] = state_dict[new_key_]
-                except:
-                    continue
+                s = key_.split('.')
+                idx = int(s[-2])
+                new_key_ = ".".join(s[:-2] + [str(int(idx + 1))] + [s[-1]])
+                state_dict[key_] = state_dict[new_key_]
 
```

Comment:
It's not a great practice but we do need this guard because in multiple usecases, this string s[-2] won't be a number to convert, it avoids throwing up an error.

Reply:
So unless there is better solution, we need to keep this try statement.